docs/source/setup.md

```diff
@@ -231,3 +231,105 @@ Possible solutions for the same problem can be found at [link]
 [link]: http://stackoverflow.com/a/28938258/2534102
 [backend]: http://127.0.0.1:8000
 [frontend]: http://127.0.0.1:8888
+
+## Windows Installation Instructions
+
+### Step 1: Install prerequisites
+
+* Install git from https://git-scm.com/downloads
+
+* Install postgres from https://www.postgresql.org/download/windows/
```

Comment:
Please add version here.

Reply:
@RishabhJain2018 I think that we can leave out the version since it's common along all the OS and is already mentioned at the very start of the Readme.

Reply:
Yeah I agree @the-dagger

Reply:
@the-dagger @xamfy: I don't agree because the developer definitely needs the Postgres version >=9.4 otherwise there is no support for `JSONField` in previous versions of Postgres and that will create the problem. @xamfy can you please add the version `>=9.4`